src/pages/PlanPage/subRoutes/InvoiceDetailsPage/sections/InvoiceHeader.tsx

```diff
@@ -32,20 +38,23 @@ function InvoiceHeader({ invoice, accountDetails }) {
         <div>
           <p className="font-semibold">Codecov</p>
           <address className="not-italic text-gray-800">
-            Codecov LLC
+            Functional Software, dba Sentry
             <br />
-            9450 SW Gemini Drive
+            45 Fremont St.
```

Comment:
should we include 8th floor here too? Going off of this https://sentry.io/privacy/